pom.xml

```diff
@@ -150,7 +150,6 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-release-plugin</artifactId>
-                <version>2.5.3</version>
```

Comment:
Does this change mean that it will always be the most recent version?

Reply:
I was trying to suppress a warning. I think the version is imposed by another plugin version and this was colliding.